src/components/App.js

```diff
@@ -1,10 +1,35 @@
 import React from 'react'
-import '../App.css'
+import BookList from '../containers/BookList'
+import BookForm from '../containers/BookForm'
+import Navbar from '../components/Navbar'
+import Loader from './loader'
+import { fetchBooks } from '../actions'
+import { connect } from 'react-redux'
```

Comment:
I won't be particular about this but I like you to note that since we are connecting `App` into the store we should have put `App` into the `containers` folder. But then again, I don't buy that concept of having to separate components into two folders. So just saying, haha, based on what the instructions said.